internal/cmd/ocean/spark_connect.go

```diff
@@ -119,6 +120,23 @@ func (x *CmdSparkConnect) run(ctx context.Context) error {
 
 	log.Infof("Starting websocket server on address %s", socketConnection.RemoteAddr().String())
 
+	// Start ping ticker to keep connection alive
+	pingTicker := time.NewTicker(5 * time.Second)
+	go func() {
+		for {
+			select {
+			case <-ctx.Done():
+				return
+			case <-pingTicker.C:
+				if err := socketConnection.WriteMessage(websocket.PingMessage, nil); err != nil {
+					log.Errorf("Failed to send ping: %v", err)
+					return
```

Comment:
Should this perhaps just be best effort, and in case a package is not sent properly we just log the error and try again, returning will kill the go routine but other things are still going to run.